core/modules/filter/filter.module

```diff
@@ -585,7 +585,7 @@ function filter_format_allowed_html($format) {
     }
   }
 
-  // Simplification: if we have both an (intersected) allowlist and a (unioned)
+  // Simplification: if we have both an (intersected) allowlist and a (union'ed)
```

Comment:
```suggestion
  // Simplification: if we have both an (intersected) allowlist and a (unioned)
```
The apostrophe is used to indicate possessive case, contractions, and omitted letters. It's not used to turn a noun into a verb.

Reply:
That would flag the word as unknown unfortunately. There's other instances in core where we spell it like `UNIONed`, which doesn't trip CSpell. Would that be acceptable here?

Reply:
Since we're intentionally using "union" as a verb, I'd first advocate adding it to the dictionary; our usage is correct and intentional, and so that's what the dictionary should reflect.

If that's not possible (i.e., strong pushback against it), then I think "union'ed" is more readable than "UNIONed." But this is (to my mind) a distant second choice.

(This bit of verb pedantry coming from the coiner of "enhappify.")

Reply:
...so this ^^ flags the word in CSpell. This (`UNIONed`) doesn't: https://github.com/backdrop/backdrop/blob/1.x/core/includes/database/select.inc#L435

Reply:
...I've added a comment in the main issue to discuss and decide on this.

Reply:
Fixed with https://github.com/backdrop/backdrop/pull/4575/commits/15015308052afc2efc28fc80fe697de2c7c2e769

See discussion over at https://github.com/backdrop/backdrop-issues/issues/6255#issuecomment-1825012854